src/offspot_demo/setup.py

```diff
@@ -1,2 +1,143 @@
+import shutil
+import subprocess
+import sys
+from time import sleep
+
+from offspot_demo.constants import (
+    DOCKER_COMPOSE_SYMLINK_PATH,
+    FQDN,
+    JINJA_ENV,
+    SRC_PATH,
+    STARTUP_DURATION,
+    SYSTEMD_UNIT_NAME,
+    SYSTEMD_UNIT_PATH,
+)
+
+
+def render_maint_caddyfile():
+    """Render the maintenance Caddyfile to customize local stuff"""
+    print("Rendering maintenance Caddyfile")
+    with open(SRC_PATH / "maint-compose/Caddyfile", "w") as fh:
+        fh.write(
+            JINJA_ENV.get_template("maint-compose/Caddyfile.tmpl").render(fqdn=FQDN)
+        )
+
+
+def render_maint_docker_compose():
+    """Render the maintenance docker-compose to customize local stuff"""
+    print("Rendering maintenance docker-compose")
+    with open(SRC_PATH / "maint-compose/docker-compose.yml", "w") as fh:
+        fh.write(
+            JINJA_ENV.get_template("maint-compose/docker-compose.tmpl.yml").render(
+                src_path=SRC_PATH
+            )
+        )
+
+
+def install_symlink():
+    """Install docker-compose symlink
+
+    Will be switched between maint and prod, but for now point to maint
+    """
+    print("Installing docker-compose symlink")
+    DOCKER_COMPOSE_SYMLINK_PATH.unlink(missing_ok=True)
+    DOCKER_COMPOSE_SYMLINK_PATH.symlink_to(
+        SRC_PATH / "maint-compose/docker-compose.yml"
```

Comment:
When using the `/` operator instead of `.joinpath()`, I like to use it constantly for all parts. It's just a preference but looking at this line disturbs me at first.